pyphare/pyphare/core/gridlayout.py

```diff
@@ -62,7 +62,7 @@ def __init__(self):
         self.centerZ = yee_centering["z"]
 
 
-def yeeCoordsFor(origin, nbrGhosts, dl, nbrCells, qty, direction, withGhosts=False):
+def yeeCoordsFor(origin, nbrGhosts, dl, nbrCells, qty, direction, withGhosts :int =False):
```

Comment:
`int =False`  -> `int = -1` ?

Reply:
or is it a bool?

Reply:
don't know how this got here, removed :int